lib/bokken_web/controllers/organizer_controller.ex

```diff
@@ -5,13 +5,14 @@ defmodule BokkenWeb.OrganizerController do
   alias Bokken.Accounts.Organizer
 
   action_fallback BokkenWeb.FallbackController
+  defguard is_organizer(conn) when conn.assigns.current_user.role === :organizer
```

Comment:
We should probably create a file just for these guards, otherwise we are going to end up with a lot of duplicated code